evap/results/exporters.py

```diff
@@ -77,6 +77,34 @@ def deviation_to_style(self, deviation, total=False):
             style_name += "_total"
         return style_name
 
+    def filter_text_and_heading_questions(self, questions):
+        filtered_questions = []
+        current_heading_question = None
+        questions_in_current_heading = []
+
+        for question in questions:
+            if question.is_heading_question:
+                if questions_in_current_heading:
+                    if current_heading_question:
+                        filtered_questions.append(current_heading_question)
+                    filtered_questions += questions_in_current_heading
+
+                current_heading_question = question
+                questions_in_current_heading = []
```

Comment:
this can possibly be simplified by using a simple loop with a counter and checking the current element and the next element, if they are both heading questions, then do not use the first question.